config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :merchants, only: [:index, :show]
+  get "/merchants/:merchant_id/items/:item_id", to: "items#show"
+  resources :items, only: [:index, :show]
```

Comment:
I think that this would have produced very similar routes:
```
resources :merchants, only: [:index, :show] do
  resources :items, only: [:show]
end
resources :items, only: [:index, :show]
```